extensions/stock-api-fastapi/manifest.json

```
@@ -0,0 +1,42 @@
+{
+  "version": 1,
+  "locale": "en_US.UTF-8",
+  "metadata": {
+    "appmode": "python-fastapi",
+    "entrypoint": "main"
+  },
+  "extension": {
+    "name": "stock-api-fastapi",
+    "title": "FastAPI Stock Pricing Service",
+    "description": "APIs are a great way for software engineering teams to use your models without translating them into different languages",
```

Same as the other review.

```suggestion
    "description": "A simple example API, showing how you can turn your Python models into production services that other tools and teams can use without translating them into different languages.",
```

---

Opting to skip the renaming for now, we will revisit them all prior to Gallery release.